02-Servicio_completo/src/main/kotlin/rest/controller/DemographicDataController.kt

```diff
@@ -1,14 +1,23 @@
 package rest.controller
 
 import io.javalin.http.Context
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
 import rest.dto.toDto
 import service.DemographicDataService
 
 class DemographicDataController(
-        private val demographicData: DemographicDataService
+    private val demographicData: DemographicDataService
```

Comment:
could we rename this var according its function? Ex. demographicDataService / service, etc.